### ISSUE TITLE
Issue Genetic Mytosis and Animals

### ISSUE THREAD

**Issue by vegapnk** [vegapnk/RJW-Genes#95]
I got the following on Discord: 

```
Exception in SetupToils for pawn Kir driver=JobDriver_BestialityForMale (toilIndex=-1) driver.job=(Bestiality (Job_1025367) A = Thing_Squirrel37631 Giver = ThinkNode_QueuedJob [workGiverDef: null])
System.MissingMethodException: bool RimWorld.Pawn_GeneTracker.HasActiveGene(Verse.GeneDef)
[Ref F5A69714]
 at RJW_Genes.Patch_OrgasmMytosis.Postfix (rjw.JobDriver_Sex __instance) [0x0000d] in <a29571054ff24f6e817467bcf2a38d74>:0 
 at rjw.JobDriver_Sex.SetupOrgasmTicks () [0x00064] in <677f214841e54affa3a84eda322bf370>:0 
     - POSTFIX rjw_genes: Void RJW_Genes.Patch_OrgasmMytosis:Postfix(JobDriver_Sex __instance)
 at rjw.JobDriver_Sex.setup_ticks () [0x00006] in <677f214841e54affa3a84eda322bf370>:0 
     - POSTFIX rjwanim: Void Rimworld_Animations.HarmonyPatch_JobDriver_Sex:Postfix(JobDriver_Sex __instance)
     - POSTFIX Privacy_Please: Void Privacy_Please.HarmonyPatch_JobDriver_Sex_setup_ticks:Postfix(JobDriver_Sex& __instance)
 at rjw.JobDriver_BestialityForMale+<MakeNewToils>d__1.MoveNext () [0x00069] in <677f214841e54affa3a84eda322bf370>:0 
 at Verse.AI.JobDriver.SetupToils () [0x00057] in <9b17790b066e46d08be4026d65926375>:0 
     - POSTFIX roolo.RunAndGun: Void RunAndGun.Harmony.JobDriver_SetupToils:Postfix(JobDriver __instance, List`1 ___toils) 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 39)
```
[output_log-1.txt](https://github.com/user-attachments/files/15717569/output_log-1.txt)

Seems like a missing null-check on bestiality. 
[Relevant File](https://github.com/vegapnk/RJW-Genes/blob/main/Source/Genes/Special/Patches/Patch_OrgasmMytosis.cs)

**Comment by vegapnk** [comment]
It could also be related to #104 , because the `HasActiveGene` is mentioned.

**Comment by vegapnk** [comment]
Should be fixed with 2.1.0
